File: exercise01/src/Exercise.java
import java.util.GregorianCalendar;

public class Exercise {

    public static void main(String[] args) {
        JavaPerson john = new JavaPerson(1L, "Mr.", "John", "Blue", new GregorianCalendar(1977,9,3));
        JavaPerson jane = new JavaPerson(1L, "Mrs.", "Jane", "Green", null);

        System.out.println(john + "'s age is " + john.getAge());
        System.out.println(jane + "'s age is " + jane.getAge());

        int age = JavaPerson.getAge(new GregorianCalendar(1988, 5, 3));
        System.out.println("The age of someone born on 3rd May 1988 is " + age);
    }
}

File: lessons02/src/Exercise.kt
import java.util.GregorianCalendar

class Exercise

fun main(args: Array<String>) {
    val john = KotlinPerson(1, "Mr.", "John", "Blue", GregorianCalendar(1977, 9, 3))
    val jane = KotlinPerson(1, "Mrs.", "Jane", "Green", null)

    println(john.toString() + "'s age is " + john.age)
    println(jane.toString() + "'s age is " + jane.age)

    val age = KotlinPerson.getAge(GregorianCalendar(1988, 5, 3))
    println("The age of someone born on 3rd May 1988 is $age")
}
